src/CalendarAvailability.vue

```diff
@@ -100,6 +100,10 @@ export default {
 			type: String,
 			required: true,
 		},
+		l10nWeekDayListLabel: {
+			type: String,
+			default: 'List of weekdays',
```

Comment:
Because `<UL>` semantics already has "List" semantics. 

Otherwise, a screen reader will read it as something like "List of weekdays list with 7 items".

```suggestion
			default: 'Weekdays',
```